.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide/first-contribution.md#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
+2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
+https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
+3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
+4. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+5. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+-->
+
+#### What type of PR is this?
+
+<!--
+Add one of the following kinds:
+/kind bug
+/kind cleanup
+/kind documentation
+/kind feature
+
+Optionally add one or more of the following kinds if applicable:
+/kind api-change
+/kind deprecation
+/kind failing-test
+/kind flake
+/kind regression
+-->
+
+#### What this PR does / why we need it:
+
+#### Which issue(s) this PR fixes:
+<!--
+*Automatically closes linked issue when PR is merged.
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
+_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
+-->
+Fixes #
+
+#### Special notes for your reviewer:
+
+#### Does this PR introduce a user-facing change?
+<!--
+If no, just write "NONE" in the release-note block below.
+If yes, a release note is required:
+Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+-->
+```release-note
+
+```
```

Comment:
Do we need this? I'm not sure if the bot supports the release note generation feature on sigs repo.

Reply:
This is synced with the k/k repo.

Reply:
At least better to know the impact of the PR? If you disagree, I can remove.

Reply:
I checked the behavior of the bot in:
https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/pull/4

It even doesn't add `release-note` related labels. 

But, I agree that we should have a section to explain what's the expected user facing changes.
So, can you simplify this section like:

```md
#### Does this PR introduce a user-facing change?
<!--
Enter a brief explanation of what are the expected user-facing changes by this PR.
-->

```

Reply:
We can generate release note following https://github.com/kubernetes/release/tree/master/cmd/release-notes
So let keep the same with k/k to work well with release-note cli.

Reply:
Oh, sounds great. Yep, then let's go with it.